index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>International Javascript Conference</title>
+    <script src="https://use.fontawesome.com/54aef7fad9.js"></script>
+    <script src="https://kit.fontawesome.com/e99fae66d4.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!--  HEADER SECTION  -->
+  <header>
+    <div class="top">
+        <ul class="top-header">
+            <li><i class="fa fa-facebook" aria-hidden="true"></i></li>
+            <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+        </ul>
+    </div>
+    <nav class="navbar">
+        <i class="fa fa-bars" aria-hidden="true"></i>
+        <a href="index.html"><img src="images/logo.png" alt="Logo" class="logo"></a>
+        <ul class="nav-links">
+            <li><a href="about.html">About</a></li>
+            <li><a href="#program">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#sponsors">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li><button type="button">JSI Conference</button></li>
+        </ul>
+        <div class="mob-menu hidden">
+            <i class="fa fa-times" aria-hidden="true" id="close"></i>
+            <ul class="menu-link">
+                <li><a href="about.html" class="links">About</a></li>
+                <li><a href="#program" class="links">Program</a></li>
+                <li><a href="#" class="links">Join</a></li>
+                <li><a href="#sponsors" class="links">Sponsor</a></li>
+                <li><a href="#" class="links">News</a></li>
+                <li><button type="button">JSI Conference</button></li>
+            </ul>
+        </div>
+    </nav>
+  </header>
+  <main>
+      <!--  INTRODUCTION SECTION  -->
+      <section class="intro">
+          <div>
+             <h2>"Hello! Sharing world"</h2>
+             <h1>International Javascript Conference 2022</h1>
+          </div>
+          <article>
+              <p class="intro-txt">International javascript Conference is a summit that talks about the latest innovations in javascript.This is to help broaden the knowledge of web developers and javascript enthusiast</p>
+          </article>
+          <div>
+              <h3>From April 25th ~ 28th, 2022</h3>
+              <p>@ The O2 Arena. Peninsula Square, London SE10 0DX, United Kingdom</p>
+          </div>
+      </section>
+      <!--  MAIN PROGRAM SECTION  -->
```

Comment:
- [ ] In the `main program` section there is some gap between the boxes in your design but in the original, there is not. Take a look at below 👇 
- Yours 👇 
![main-program-st](https://user-images.githubusercontent.com/67087664/147965648-cd25dcf2-26af-44d3-83e4-92e2502f926e.png)
- Original 👇 
![main-program](https://user-images.githubusercontent.com/67087664/147965739-1f8170cb-7742-4f53-ac56-cf78bf8b61b8.png)